lib/modules/onc_program_module.yml

```diff
@@ -141,6 +141,7 @@ test_sets:
           These resources will not be tested for FHIR search support.
         lock_variables: 
           - token
+          - attachment_requires_token
```

Comment:
Why add this input if it is locked?

Reply:
We think though server can support self-signed urls for attachment and we programed our testing to take this boolean option into logic. Since there is still debating if self-signed urls has the same effects as OAuth token, we decide to disable that at this release. If there is feedback from implementer community indicating preference of self-signed urls, it is easier to change this configuraiton than actual coding.